cypress/e2e/supervision/deputies/add-deputy.cy.js

```diff
@@ -0,0 +1,93 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createAClient();
+  cy.get("@clientId").then((clientId) => cy.createOrderForClient(clientId));
+  cy.get("@clientId").then((clientId) => {cy.visit(`/supervision/#/clients/${clientId}`)});
+});
+
+describe(
+  "Create deputy for client",
+  { tags: ["@supervision", "@deputy", "supervision-core"] },
+  () => {
+    it("Given I'm a Case Manager on Supervision, I can add a deputy to a clients order using only mandatory fields. This deputy will then be selected as main correspondant and fee payer.", () => {
+      cy.get("@clientId").then((clientId) => {
+        cy.searchForADeputyToReachAddADeputyPage();
+        cy.contains("Lay").click();
+        cy.get('[label="First name"]').type("Patrick");
+        cy.get('[label="Last name"]').type("Star");
+        cy.contains("Save & continue").click();
```

Comment:
Best to avoid hooks in tests based on the label alone as that can pick up unrelated content further up the DOM (if there's a hidden element with that label or multiple buttons or headers, that kind of thing. So better to combine with an element name (button/a) or some more specific hook.